src/vsr/superblock.zig

```diff
@@ -79,17 +80,20 @@ pub const SuperBlockHeader = extern struct {
     /// A monotonically increasing counter to locate the latest superblock at startup.
     sequence: u64,
 
-    /// Protects against writing to or reading from the wrong data file.
-    cluster: u128,
-
     /// The checksum of the previous superblock to hash chain across sequence numbers.
     parent: u128,
+    parent_padding: u128 = 0,
```

Comment:
Is it intentional that we sometimes use `checksum_padding` and sometimes `checksum_reserved`?

Reply:
No, not intentional. I'll change them all to "padding".

Reply:
Related, we could do `nonce_reserved` instead of `reserved_nonce`.